x-pack/solutions/search/plugins/search_synonyms/public/components/synonym_sets/synonym_sets.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SynonymsGetSynonymsSetsSynonymsSetItem } from '@elastic/elasticsearch/lib/api/types';
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiButtonIcon,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiPopover,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { DEFAULT_PAGE_VALUE, paginationToPage } from '../../../common/pagination';
+import { useFetchSynonymsSets } from '../../hooks/use_fetch_synonyms_sets';
+
+export const SynonymSets = () => {
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(DEFAULT_PAGE_VALUE.size);
+  const { from } = paginationToPage({ pageIndex, pageSize, totalItemCount: 0 });
+  const { data: synonyms } = useFetchSynonymsSets({ from, size: pageSize });
+  const [isPopoverOpen, setIsPopoverOpen] = React.useState(false);
+
+  if (!synonyms) {
+    return null;
+  }
+
+  const pagination = {
+    initialPageSize: 10,
+    pageSizeOptions: [10, 25, 50],
+    ...synonyms._meta,
+    pageSize,
+    pageIndex,
+  };
+  const columns: Array<EuiBasicTableColumn<SynonymsGetSynonymsSetsSynonymsSetItem>> = [
+    {
+      field: 'synonyms_set',
+      name: i18n.translate('xpack.searchSynonyms.synonymsSetTable.nameColumn', {
+        defaultMessage: 'Synonyms Set',
+      }),
+      render: (name: string) => <div data-test-subj="synonyms-set-item-name">{name}</div>,
+    },
+    {
+      field: 'count',
+      name: i18n.translate('xpack.searchSynonyms.synonymsSetTable.ruleCount', {
+        defaultMessage: 'Rule Count',
+      }),
+      render: (ruleCount: number) => (
+        <div data-test-subj="synonyms-set-item-rule-count">{ruleCount}</div>
+      ),
+    },
+    {
+      actions: [
+        {
+          render: (item: SynonymsGetSynonymsSetsSynonymsSetItem) => (
```

Comment:
Do we need a `render` here with the popover and everything done manually? You can just pass an object describing the action:
https://eui.elastic.co/#/tabular-content/tables#adding-actions-to-table

```
{
          name: 'User profile',
          description: ({ firstName, lastName }) =>
            `Visit ${firstName} ${lastName}'s profile`,
          icon: 'editorLink',
          color: 'primary',
          type: 'icon',
          enabled: ({ online }) => !!online,
          href: ({ id }) => `${window.location.href}?id=${id}`,
          target: '_self',
          'data-test-subj': 'action-outboundlink',
        }
```

Reply:
no we don't see: https://github.com/elastic/kibana/pull/207075

Once this is in I will enable the other PR. Until then I can simply delete the actions column I guess instead.